VersionControlUI/src/DiffManager.cpp

```diff
@@ -94,9 +94,11 @@ DiffSetup DiffManager::initializeDiffPrerequisites(QString oldVersion, QString n
 
 	// load newer version
 	diffSetup.newVersionManager_ = createTreeManagerFromVersion(diffSetup.repository_, diffSetup.newVersion_);
+	diffSetup.newVersionManager_->setManagerName(project_+"#newVersion");
```

Comment:
Another thing is that this commit should have ideally included the code where you use the `managerName`, cause right now you're just setting a field and it doesn't become clear why this is necessary. I'm guess further commits will make this clear.